JavaErickHernandez/Day3/Maps.java

```diff
@@ -0,0 +1,93 @@
+package bootcamp;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Maps {
+	static String numberOfInputs = "Enter the count of phone numbers: ";
```

Comment:
What is the reason for not putting these variables as "final"? if you put "final" you avoid to any other person can change this variables in other par of the code.